src/sentry/event_manager.py

```diff
@@ -1474,10 +1489,33 @@ def _save_aggregate(
             else:
                 root_hierarchical_grouphash = None
 
-            # If we still haven't found a matching grouphash, we're now safe to go ahead and create
-            # the group.
+            # If we still haven't found a matching grouphash, we're now safe to go ahead and talk to
+            # seer and/or create the group.
             if existing_grouphash is None:
-                group = _create_group(project, event, **group_creation_kwargs)
+                seer_matched_group = None
+
+                if should_call_seer_for_grouping(event, project):
+                    try:
+                        # If the `projects:similarity-embeddings-grouping` feature is disabled,
+                        # we'll still get back result metadata, but `seer_matched_group` will be None
+                        seer_response_data, seer_matched_group = get_seer_similar_issues(
+                            event, primary_hashes
+                        )
+                        event.data["seer_similarity"] = seer_response_data
+
+                        # We only want to add this data to new groups, while we're testing
+                        # TODO: Remove this once we're out of the testing phase
```

Comment:
Wouldn't we want to keep that for debugging even after testing phase?

Reply:
We might end up keeping it, but the current plan is to take it out eventually. For debugging purposes, we have the same information ~on the event~ hopefully added to the corresponding `GroupHash` record. (See https://github.com/getsentry/sentry/issues/70454.)